src/Modal/Portal.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+import cx from 'classnames';
+import Panel from '../Panel/Panel';
+import styles from "./Modal.module.scss";
+
+class Portal extends React.Component {
+  constructor() {
+    super();
+
+    this.mountNode = document.createElement('div');
+    this.mountNode.id = 'ui-modal';
+    this.mountNode.className = styles.wrapper;
+    document.body.appendChild(this.mountNode);
```

Comment:
[Comment not resolved](https://github.com/santiment/san-ui/pull/39#discussion_r264360165). Should be the same as in the example